### PR TITLE
[ja] Translate reference/glossary/cluster-architect.md

### LINKED ISSUE #45115: [ja] Translate `content/en/docs/reference/glossary/cluster-architect.md` into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate `content/en/docs/reference/glossary/cluster-architect.md` into Japanese

**Why is this needed**

There is no Japanese translation for this page

**Comments**

- File to translate: 
    - https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/cluster-architect.md
- Original:
    - https://kubernetes.io/ja/docs/reference/glossary/?all=true#term-cluster-architect

日本語に翻訳する際は、以下のドキュメントを参考にしてください。

- Japanese Localization document: https://kubernetes.io/ja/docs/contribute/localization/

### PR DESCRIPTION
**Pull request by rmiki**
add: content/ja/docs/reference/glossary/cluster-architect.md

close #45115

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
@rmiki I see the bot has added `do-not-merge/invalid-commit-message` as the [commit message](https://github.com/kubernetes/website/pull/45408/commits/e08a0c8c628249eafedadfbf11f45daf8754022a) has "close https://github.com/kubernetes/website/issues/45115" ([Blocked Keywords Guide](https://www.kubernetes.dev/docs/guide/pull-requests/#do-not-use-github-keywords-or-mentions-within-your-commit-message)). Refer bot's message below:

> The list of commits with invalid commit messages:
[e08a0c8](https://github.com/kubernetes/website/commits/e08a0c8c628249eafedadfbf11f45daf8754022a) [ja] Translate reference/glossary/cluster-architect.md

Would you be able to edit the commit message to remove the [Blocked Keyword](https://www.kubernetes.dev/docs/guide/pull-requests/#do-not-use-github-keywords-or-mentions-within-your-commit-message) in order to move this PR forward?

**Comment by rmiki** [comment]
@dipesh-rawat

Thank you for comment. I'll remove it.

File: content/ja/docs/reference/glossary/cluster-architect.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: クラスターアーキテクト
+id: cluster-architect
+date: 2018-04-12
+full_link: 
+short_description: >
+  一つ以上のKubernetesクラスターで構成されるインフラストラクチャーを設計する人
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/cluster-architect.md:7]:
```suggestion
  一つ以上のKubernetesクラスターで構成されるインフラストラクチャーを設計する人。
```

Please add "。"

File: content/ja/docs/reference/glossary/cluster-architect.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+aka: 
+tags:
+- user-type
+---
+ 一つ以上のKubernetesクラスターで構成されるインフラストラクチャーを設計する人
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/cluster-architect.md:13]:
```suggestion
 一つ以上のKubernetesクラスターで構成されるインフラストラクチャーを設計する人。
```

ditto

**Review by Okabe-Junya — CHANGES_REQUESTED** [review]
Thank you for your contribution!!

I have some nits, so please check it out :)

**Review by Okabe-Junya — APPROVED** [review]
Thank you for updating!!

/lgtm

**Review by nasa9084 — COMMENTED** [review]
Thank you for your first contribution!! :tada:
Sorry for late review... 🙇 

/approve

**Pull request merged**